packages/sdk/README.md

```diff
@@ -1,3 +1,98 @@
 # XCM-SDK
 
 ## :warning: This package is not production ready and is in heavy development!
+
+## Installation
+
+```bash
+npm i @moonbeam-network/xcm-config @moonbeam-network/xcm-sdk
+```
+
+## Usage
+
+### Init
+
+```typescript
+import { init } from '@moonbeam-network/xcm-sdk';
+
+const { moonbeam } = init();
+
+// Optionally you can provide signers or do it on get()
+const { moonbeam } = init({
+  ethersSigner,
+  polkadotSigner,
+});
+```
+
+### Accounts
+
+```typescript
+const moonbeamAddress = '0xeF46c7649270C912704fB09B75097f6E32208b85';
+const substrateAddress = '5DG5Fn3ww3KPza1RLoap6QJNzQfEvRebxypDGp35YuMX5y2K';
+```
+
+### Deposit
+
+```typescript
+// moonbeam.symbols is a list of asset symbols that you can deposit/withdraw
+// chains is a list of chains from which you can deposit asset
+const { chains, from } = moonbeam.deposit(moonbeam.symbols[0]);
+
+const { min, send } = await from(chains[0]).get(
+  moonbeamAddress,
+  substrateAddress,
+  // if you haven't provided on init
+  { polkadotSigner },
+);
+
+await send(min, (event) => console.log(event));
+```
+
+### Withdraw
+
+```typescript
+const { chains, to } = moonbeam.withdraw(moonbeam.symbols[0]);
+
+const { min, send } = await to(chains[0]).get(
+  substrateAddress,
+  // if you haven't provided on init
+  { ethersSigner },
+);
+
+await send(min, (event) => console.log(event));
+```
+
+### Get balances
+
+```typescript
+const unsubscribe = await moonbeam.subscribeToAssetsBalanceInfo(
```

Comment:
Just now coming to this realization, but maybe we should also offer a method to get balance of only one of the assets

Reply:
Yeah, I also thought about it.

Reply:
I think we should also move getMax from dapp to sdk.

Reply:
That's a good idea, should be useful to other teams

Reply:
agree